components/ai_chat/resources/page/components/conversation_list/index.tsx

```diff
@@ -123,6 +124,7 @@ function ConversationList(props: ConversationListProps) {
           const isHuman = turn.characterType === mojom.CharacterType.HUMAN
           const isAIAssistant = turn.characterType === mojom.CharacterType.ASSISTANT
           const showSiteTitle = id === 0 && isHuman && shouldSendPageContents
+          const showLongPageContentInfo = id === 1 && isAIAssistant && context.shouldShowLongPageWarning
```

Comment:
this looks flaky. we could potentially attach an ActionType for requests that have been coming from suggested questions list?